app/javascript/react/screens/App/Plan/PlanActions.js

```diff
@@ -206,3 +207,36 @@ export const downloadLogAction = task => dispatch => {
     })
   });
 };
+
+// *****************************************************************************
+// * CANCEL MIGRATION TASKS
+// *****************************************************************************
+export const cancelPlanRequestTasksAction = (url, tasks) => dispatch => {
+  const resources = tasks.map(t => ({ id: t.id }));
+  dispatch({
+    type: CANCEL_V2V_PLAN_REQUEST_TASKS,
+    payload: new Promise((resolve, reject) => {
+      API.post(url, {
+        action: 'cancel',
+        resources
+      })
+        .then(response => {
+          resolve(response);
+
+          const cancelSuccessMsg = __('Migration canceled successfully for VMs: ');
+          const vmNames = [];
+          tasks.map(task => vmNames.push(task.vmName));
+          const cancelSuccessMsgWithVMNames = `${cancelSuccessMsg} ${vmNames}`;
```

Comment:
Minor nit, but if we pass an array (with multiple entries) into the template literal, there will not be spaces after each comma

Reply:
Agreed, it might be worth doing something like this:

```
const vmNamesStr = vmNames.join(', ');
const cancelSuccessMsgWithVMNames = `${cancelSuccessMsg} ${vmNamesStr}`;
```

Reply:
Thanks for the tip.
Addressed this in the last commit